.github/workflows/application.yml

```diff
@@ -582,6 +582,14 @@ jobs:
                     name: "Behat logs (non-JS, PHP ${{ matrix.php }}, Symfony ${{ matrix.symfony }})"
                     path: etc/build/
                     if-no-files-found: ignore
+                    
+            -   
+                name: Upload varlog logs
+                uses: actions/upload-artifact@v3
+                with:
+                    name: "varlog logs (non-JS, PHP ${{ matrix.php }}, Symfony ${{ matrix.symfony }})"
+                    path: var/log/
+                    if-no-files-found: ignore
```

Comment:
I would be for reverting this change as we are not uploading logs for other builds, and I believe it is not needed